docs/release_notes.md

```diff
@@ -1,5 +1,179 @@
 #
 
+## 4.3.4
+
+Release Date: 2025-03-XX
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 4.3.4.
+
+This release ...
+
+- deprecations in support of irods 5
+- ssl verification
+- bug fixes
+- clean up
+
+This release consists of [93 commits from 7 contributors](https://github.com/irods/irods/compare/4.3.3...4.3.4) and [closed 132 issues marked for 4.3.4](https://github.com/irods/irods/issues?q=milestone%3A4.3.4).
```

Comment:
Consider changing 7 to 6. The only thing is that we provide a link to GitHub which reports 7.

From @trel:
```
$ git shortlog -ns 4.3.3...4-3-stable
    29  Kory Draughn
    23  Markus Kitsinger (SwooshyCueb)
    18  Derek D
    12  Alan King
     7  Martin Jaime Flores Jr.
     3  Martin Jaime Flores Jr
     1  Justin James
```

Reply:
I think there is precedent in using the correct number, in this case 6.    GitHub doesn't know all of Martin's secrets.

Reply:
I agree. Plus, the output you provided from `git` is now part of this PR's history.

Will change it to 6.

Reply:
Done.